src/components/20-molecules/list/story.js

```diff
@@ -0,0 +1,63 @@
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+
+export default {
+  title: 'Components/List',
+  decorators: [withKnobs],
+};
```

Comment:
You should give "usage addon" some informations about the childs. to display a good example there.

Reply:
Oh nice tipp, thx. I somehow assumed that this tab would be kinda dumb.